sdk/cosmos/tests.yml

```diff
@@ -190,7 +189,6 @@ jobs:
           DisplayName: Single Region ReadMyWrites
           OSVmImage: 'ubuntu-18.04'
           ProfileFlag: '-Pe2e'
-          AdditionalArgs: '-DargLine="-Dcosmos.directModeProtocol=Https"'
```

Comment:
ditto

Reply:
Yes, currently on V3- we are not running e2e tests on Tcp mode, which is important as compared to running e2e tests on https mode.